app/src/components/games/reefscape/ReefscapeLevels.tsx

```diff
@@ -0,0 +1,83 @@
+import {StyleSheet, Text, TouchableOpacity, View} from 'react-native';
+import React from 'react';
+import {useTheme} from '@react-navigation/native';
+import ReactNativeHapticFeedback from 'react-native-haptic-feedback';
+import {ReefscapeActionType} from './ReefscapeActions';
+
+export const ReefscapeLevels = ({
+  onSubmit,
+}: {
+  onSubmit: (level: ReefscapeActionType) => void;
+}) => {
+  const {colors} = useTheme();
+  const styles = StyleSheet.create({
+    button: {
+      backgroundColor: colors.primary,
+      alignItems: 'center',
+      justifyContent: 'center',
+      borderRadius: 8,
+      padding: 10,
+      margin: 10,
+      width: 170,
+      height: 90,
```

Comment:
having an arbitrary width and height means that this button will not look the same on all devices. pls use flexbox and dynamic syzing (% of width, etc)